core/data/mapping.go

```diff
@@ -20,6 +20,8 @@ const (
 
 	// MtObject denotes a object construction mapping
 	MtObject MappingType = 4
+
+	MTARRAY MappingType = 5
```

Comment:
@mellistibco what should be our casing for constants?  MtArray, MTARRAY, MT_ARRAY.  We are not consistent in the code. The standard library using Camel Case and I think that is the suggested way to do it.  Should we fix this to be consistent?

Reply:
yes, I think camel case is the way to go imo...

Reply:
@lixingwang Can you change MTARRAY to MtArray to be consistent with the other consts

Reply:
@fm-tibco Done